modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -337,18 +450,20 @@ public void testSendNotification() throws Exception {
 	}
 
 	private NotificationTemplate _addNotificationTemplate(
-			String body, String editorType, boolean singleRecipient,
-			Map<Locale, String> to)
+			String body, String editorType, long objectDefinitionId,
+			boolean singleRecipient, Map<Locale, String> to)
 		throws Exception {
 
-		ObjectField objectField = objectFieldLocalService.getObjectField(
-			childObjectDefinition.getObjectDefinitionId(),
-			"attachmentObjectField");
+		ObjectField objectField = objectFieldLocalService.fetchObjectField(
+			objectDefinitionId, "attachmentObjectField");
+
+		List<Long> objectId = (objectField != null) ?
```

Comment:
@rijema rename to `attachmentObjectFieldIds`

Reply:
Done!